### PR TITLE
FIX failsCounter for all kind of subscriptions

### PR DESCRIPTION
**Pull request by fgalan**

### CODE REVIEW

File: test/functionalTest/cases/3001_mqtt_alarms/mqtt_alarms_raise_and_release.test [hunk 1]
```diff
@@ -238,8 +239,8 @@ Date: REGEX(.*)
 
 
 
-05. Get sub and see status:failed, timesSent:2 and lastFailure (but no lastSuccess)
-===================================================================================
+05. Get sub and see status:failed, timesSent:1, failsCounter:2 and lastFailure (but no lastSuccess)
+===================================================================================================
 HTTP/1.1 200 OK
 Content-Length: 404
```
Comment (mapedraza) [inline test/functionalTest/cases/3001_mqtt_alarms/mqtt_alarms_raise_and_release.test:245]:
```suggestion
Content-Length: 421
```
This needs to be updated - The reason why tests are failling

Comment (fgalan) [reply test/functionalTest/cases/3001_mqtt_alarms/mqtt_alarms_raise_and_release.test]: Good catch! Thanks!

Comment (fgalan) [reply test/functionalTest/cases/3001_mqtt_alarms/mqtt_alarms_raise_and_release.test]: Fixed in ff98ebc

File: test/functionalTest/cases/3001_mqtt_alarms/mqtt_alarms_raise_and_release.test [hunk 2]
```diff
@@ -187,8 +187,8 @@ Date: REGEX(.*)
 
 
 
-03. Get sub and see status:failed, timesSent:1 and lastFailure (but no lastSuccess)
-===================================================================================
+03. Get sub and see status:failed, timesSent:1, failsCounter:1 and lastFailure (but no lastSuccess)
+===================================================================================================
 HTTP/1.1 200 OK
 Content-Length: 404
```
Comment (mapedraza) [inline test/functionalTest/cases/3001_mqtt_alarms/mqtt_alarms_raise_and_release.test:193]:
```suggestion
Content-Length: 421
```

Comment (fgalan) [reply test/functionalTest/cases/3001_mqtt_alarms/mqtt_alarms_raise_and_release.test]: Fixed in ede729b

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**